tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -116,7 +110,20 @@ def is_single_input(input: [tuple, list]):
     return True
 
 
-def input_to_input_cut_info(input: [str, tuple, list]):
+def parse_inputs(inputs: str):
+    inputs_list = []
+    # Split to list of string
+    for input_value in split_inputs(inputs):
+
+        # Parse string with parameters for single input
+        node_name, shape = parse_input_value(input_value)
+        # pylint: disable=no-member
+        inputs_list.append((node_name, shape))
+    return inputs_list
+
+
+def input_to_input_cut_info(input: [dict, tuple, list]):
```

Comment:
This method still parses input_value in 140 line. 
Looks like we need to remove lines 134-144.

Reply:
oouups, i actually removed them, but here they left by mistake. Will remove them

Reply:
Corrected my comment. Let's not remove them, but replace with the suggested code. As single string should be treated as single node name.

Reply:
removed this lines.
discussed offline, after all i decided to leave checking if it's a string outside of `input_to_input_cut_info`, e.g.
https://github.com/openvinotoolkit/openvino/blob/372009b6f02aa6300c616895bd45ce107c46f849/tools/ovc/openvino/tools/ovc/convert_impl.py#L97-L101